test/integration/targets/iosxr_acl_interfaces/tests/cli/empty_config.yaml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Can we have one for 'parsed' state as well ?

Reply:
Sure. I'll add that too.